tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects.ts

```diff
@@ -94,7 +94,10 @@ import {
   getPollSilenceTimeMs,
   getSourceFileListLoaded,
 } from '../store/debugger_selectors';
-import {beginEndRangesInclude} from '../store/debugger_store_utils';
+import {
+  beginEndRangesInclude,
+  stackFrameAsArray2StackFrame,
```

Comment:
`stackFrameAsArray2StackFrame` -> `stackFrameAsArrayToStackFrame`? `2`, to me, can mean either: (1) version 2 (2) `to` or (3) two dimensions of the array